tensorflow/contrib/factorization/python/ops/gmm.py

```diff
@@ -102,7 +102,12 @@ def score(self, input_fn=None, batch_size=None, steps=None):
     results = self.evaluate(input_fn=input_fn, batch_size=batch_size,
                             steps=steps)
     return np.sum(results[GMM.SCORES])
-
+  
+  def weights(self):
+    """Returns the cluster weights."""
+    return checkpoint_utils.load_variable(
+        self.model_dir, gmm_ops.GmmAlgorithm.CLUSTERS_WEIGHT)
```

Comment:
not yet but i can put one in [gmm_test.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/factorization/python/ops/gmm_test.py)